### PR TITLE
Updated docs examples/recommendations to use findDOMNode instead of getDOMNode

### PR DESCRIPTION
**Pull request by jsfb**
Updated docs examples/recommendations to use findDOMNode instead of getDOMNode

### CODE REVIEW

**Comment by jsfb** [comment]
Code change was already merged here: https://github.com/facebook/react/pull/2646

**Comment by cody** [comment]
Maybe add it to the Top-Level API doc as well?

**Comment by zpao** [comment]
Yea, let's add the toplevel doc.

File: docs/docs/ref-03-component-specs.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Use this as an opportunity to operate on the DOM when the component has been upd
 
 > Note:
 >
-> Prior to v0.9, the DOM node was passed in as the last argument. If you were using this, you can still access the DOM node by calling `this.getDOMNode()`.
+> Prior to v0.9, the DOM node was passed in as the last argument. If you were using this, you can still access the DOM node by calling `React.findDOMNode(this)`.
```
Comment (spicyj) [inline docs/docs/ref-03-component-specs.md]: Let's kill this note completely.

**Comment by zpao** [comment]
Alright, lets address the notes and add this to the top level API docs, then we can merge this.

**Comment by jsfb** [comment]
@zpao good to go?

File: docs/docs/ref-01-top-level-api.md [hunk 2]
```diff
@@ -110,6 +110,15 @@ boolean isValidElement(* object)
 Verifies the object is a ReactElement.
 
 
+### React.findDOMNode
+
+```javascript
+DOMElement findDOMNode(ReactComponent component)
+```
+
+Finds the native DOM element, if any, associated with the given ReactComponent.
```
Comment (spicyj) [inline docs/docs/ref-01-top-level-api.md]: Can you just copy the text from here? http://facebook.github.io/react/docs/component-api.html#getdomnode

Maybe worth adding a note there too saying that React.findDOMNode is preferred.

File: docs/docs/ref-03-component-specs.md (hunk 1 above)
Comment (spicyj) [inline docs/docs/ref-03-component-specs.md]: (ping)

**Comment by jsfb** [comment]
@spicyj better?

**Comment by spicyj** [comment]
@jsfb shipit

**Pull request merged**